app/controllers/api/v1/service_offerings_controller.rb

```diff
@@ -36,6 +48,13 @@ def params_for_order
         ).to_h
       end
 
+      def params_for_applied_inventories
+        @params_for_applied_inventories ||= params.permit(
+          :service_offering_id,
+          :service_parameters => {}
```

Comment:
Why do you need service_parameters?

Reply:
it should contain "Prompt on launch" inventory from catalog. As I posted comment with presentation to "Tag based Approvals" doc, computing of applied inventories without knowledge of user input should return wrong inventories list. 
So I rather prepared full version of operation now. It means that it doesn't fail if Approval doesn't provide it, but results won't be correct.